NebulaNetwork/PacketProcessors/Factory/Belt/BeltReverseProcessor.cs

```diff
@@ -1,47 +1,62 @@
 ﻿#region
 
+using System;
 using NebulaAPI;
 using NebulaAPI.Networking;
 using NebulaAPI.Packets;
+using NebulaModel.Logger;
 using NebulaModel.Packets.Factory.Belt;
+using NebulaWorld;
 
 #endregion
 
 namespace NebulaNetwork.PacketProcessors.Factory.Belt;
 
 [RegisterPacketProcessor]
-internal class BeltReverseProcessor : BasePacketProcessor<BeltReversePacket>
+internal class BeltReverseProcessor : BasePacketProcessor<BeltReverseRequest>
 {
-    public override void ProcessPacket(BeltReversePacket packet, INebulaConnection conn)
+    public override void ProcessPacket(BeltReverseRequest packet, INebulaConnection conn)
     {
         var factory = GameMain.galaxy.PlanetById(packet.PlanetId).factory;
         if (factory == null)
         {
             return;
         }
 
+        if (IsHost)
+        {
+            var starId = packet.PlanetId / 100;
+            Multiplayer.Session.Server.SendPacketToStar(packet, starId);
+        }
+
         using (NebulaModAPI.MultiplayerSession.Factories.IsIncomingRequest.On())
         {
             NebulaModAPI.MultiplayerSession.Factories.EventFactory = factory;
             NebulaModAPI.MultiplayerSession.Factories.TargetPlanet = packet.PlanetId;
-            if (NebulaModAPI.MultiplayerSession.LocalPlayer.IsHost)
-            {
-                // Load planet model
-                NebulaModAPI.MultiplayerSession.Factories.AddPlanetTimer(packet.PlanetId);
```

Comment:
Is this not needed anymore?

Reply:
Currently the function that use `planet.physics` is `CargoTraffic.AlterBeltConnections()`, which the part is guarded by `this.isLocalLoaded (= this.planet.factoryLoaded)`. Therefore I think it's safe to remove the PlanetTimer, but I can add back if it is more robust.

Reply:
Hm, I think originally it wasn't guarded because I believe I added this due to errors that were occurring, but with this guard then the belt connections will not be modified if the planet is not loaded right? So we would still want to load it?

Reply:
The guarded part is about modifying rendering and collider variables, if my understanding is correct.
But I'll add back just in case.